var/spack/repos/builtin/packages/glvis/package.py

```diff
@@ -0,0 +1,99 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Glvis(Package):
+    """GLVis: an OpenGL tool for visualization of FEM meshes and functions"""
+
+    homepage = "http://glvis.org/"
+    url      = "https://github.com/GLvis/glvis/archive/v3.3.tar.gz"
+
+    version('3.3', '95fdd1f3b791fb3dcef541daae2ed0b8')
```

Comment:
Can we use the goo.gl shortlinks here as in mfem:

https://github.com/spack/spack/blob/af06ea0e748bf4dce52b235bdb89da47fd34613b/var/spack/repos/builtin/packages/mfem/package.py#L54-L57

You can see all the short links here: http://glvis.org/download

For glvis-3.3, the short link is: `http://goo.gl/C0Oadw`.

cc: @v-dobrev, @acfisher

Reply:
@tzanio @v-dobrev @acfisher Done.